eng/Version.Details.xml

```diff
@@ -35,46 +35,6 @@
     <Dependency Name="Microsoft.WindowsAppSDK" Version="0.0.1">
       <Uri>https://dev.azure.com/microsoft/ProjectReunion/_git/ProjectReunionInternal</Uri>
     </Dependency>
-    <Dependency Name="Microsoft.AspNetCore.App.Runtime.win-x64" Version="6.0.3" CoherentParentDependency="Microsoft.Dotnet.Sdk.Internal">
-      <Uri>https://dev.azure.com/dnceng/internal/_git/dotnet-aspnetcore</Uri>
-      <Sha>c911002ab43b7b989ed67090f2a48d9073d5118d</Sha>
-    </Dependency>
-    <Dependency Name="Microsoft.AspNetCore.Authorization" Version="6.0.3" CoherentParentDependency="Microsoft.Dotnet.Sdk.Internal">
-      <Uri>https://dev.azure.com/dnceng/internal/_git/dotnet-aspnetcore</Uri>
-      <Sha>c911002ab43b7b989ed67090f2a48d9073d5118d</Sha>
-    </Dependency>
```

Comment:
If this is removed here, what will update the `$(MicrosoftAspNetCoreAuthorizationPackageVersion)` property going forward? This is basically opting out of Maestro.

Should we instead keep using Maestro and setup subscriptions like:
```
> darc add-subscription --channel ".NET 6" --target-branch main --source-repo https://github.com/dotnet/aspnetcore --target-repo https://github.com/dotnet/maui
```
I already see 6.0.5 builds on this channel.

Maybe we should chat on Teams what you're wanting to do, thanks!

Reply:
Subscription created & triggered;

```
https://github.com/dotnet/aspnetcore (.NET 6) ==> 'https://github.com/dotnet/maui' ('main')
  - Id: f048ef0c-9fda-42a1-fd39-08da1d875c10
  - Update Frequency: EveryDay
  - Enabled: True
  - Batchable: False
  - PR Failure Notification tags:
  - Merge Policies: []
```